server_admin/topics/users/attributes.adoc

```diff
@@ -1,11 +1,18 @@
+[configuring-user-attributes]
+= Configuring user attributes
 
-=== User Attributes
+In addition to adding basic user metadata, such as names and email, you can also store arbitrary user attributes.  
 
-Beyond basic user metadata like name and email, you can store arbitrary user attributes.  Choose a user to manage
-then click on the `Attributes` tab.
+.Prerequisite
+* You are logged into the correct realm.
+
+.Procedure
+. Choose a user to manage.
+. Click on the *Attributes* tab.
+. Enter the attribute name and value in the empty fields.
+. Click *Add* to create a new field.
```

Comment:
I would say "attribute" not "field."  Also, SSO documentation has a lot of Create a New...phrases, which is redundant.  Please change this to "..to create the attribute."

Reply:
I would forgo it completely. The title should say that you are creating an attribute.